src/components/SideBar/DesktopSideBar.tsx

```diff
@@ -66,13 +83,21 @@ function DesktopSideBar({
             goalData={goalData}
             toggleSideBar={toggleSideBar}
             handleShowTodoModal={handleShowTodoModal}
+            handleShowDeletePopup={handleShowDeletePopup}
```

Comment:
[Change request] 
```suggestion
            onShowTodoModal={handleShowTodoModal}
            onShowDeletePopup={handleShowDeletePopup}
```
저희 컨벤션에 맞춰서 수정해주세요~ 다른 곳도 부탁드려용

리액트 공식문서
> 관습적으로 이벤트 핸들러 prop의 이름은 on으로 시작하여 대문자 영문으로 이어집니다